rust/operator-binary/src/druid_controller.rs

```diff
@@ -711,6 +712,21 @@ fn build_rolegroup_config_map(
                         conf.insert(S3_ENDPOINT_URL.to_string(), Some(endpoint));
                     }
 
+                    if conn.credentials.is_some() {
+                        conf.insert(
+                            S3_ACCESS_KEY.to_string(),
+                            Some(format!(
+                                "${{file:UTF-8:{S3_SECRET_DIR_NAME}/{SECRET_KEY_S3_ACCESS_KEY}}}"
+                            )),
+                        );
+                        conf.insert(
+                            S3_SECRET_KEY.to_string(),
+                            Some(format!(
+                                "${{file:UTF-8:{S3_SECRET_DIR_NAME}/{SECRET_KEY_S3_SECRET_KEY}]]"
```

Comment:
This seems to be a type and should probably read `"${{file:UTF-8:{S3_SECRET_DIR_NAME}/{SECRET_KEY_S3_SECRET_KEY}}}"`

Reply:
Thx, fixed in 550bd69